CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.1)
+project(pybind11_opencv)
+
+# Specify C++ Standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED on)
+
+find_package(catkin REQUIRED COMPONENTS
```

Comment:
this would need repackaging pretty soon but why not

Reply:
I know, I need it for some refactoring, though, which I want to finish before starting with ROS 2. As it's a very simple package, hopefully the migration will be easy.